CHANGELOG.md

```diff
@@ -4,6 +4,116 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.26.0] - 2023-07-28
+
+## 🚀 Features
+
+### Add coprocessor metrics ([PR #3483](https://github.com/apollographql/router/pull/3483))
+
+Introduces a new metric for the router:
+
+```
+apollo.router.operations.coprocessor
+```
+
+It has two attributes:
+
+```
+coprocessor.stage: string (RouterRequest, RouterResponse, SubgraphRequest, SubgraphResponse)
+coprocessor.succeeded: bool
+```
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3483
+
+### Constrain APOLLO_ROUTER_LOG and --log global levels to the router ([Issue #3474](https://github.com/apollographql/router/issues/3474))
+
+`APOLLO_ROUTER_LOG` and `--log` now implicitly set a filter constraining the logging to the `apollo_router` module, simplifying the debugging experience for users.
+
+For advanced users `RUST_LOG` can be used for standard log filter behavior.
+
+Thus:
+
+```
+RUST_LOG=apollo_router=warn
+--log warn
+APOLLO_ROUTER_LOG=warn
+```
+
+are equivalent with all three statements resulting in `warn` level logging for the router.
+
+For more details, read the logging configuration documentation.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3477
+
+### Add support for PodDisruptionBudget to helm chart ([Issue #3345](https://github.com/apollographql/router/issues/3345))
+
+A [PodDisuptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) may now be specified for your router to limit the number of concurrent disruptions.
+
+Example Configuration:
+
+```yaml
+podDisruptionBudget:
+  minAvailable: 1
+```
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3469
+
+## 🐛 Fixes
+
+### Fix the pdb template ([PR #3497](https://github.com/apollographql/router/pull/3497))
+
+Add the required `matchLabels:` attribute.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3497
+
+### Don't hide `--dev` from `--help` ([Issue #2705](https://github.com/apollographql/router/issues/2705))
+
+Display documentation about `--dev` when launching the router with `--help` argument.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/3479
+
+### Fix prometheus statistics issues with _total_total names([Issue #3443](https://github.com/apollographql/router/issues/3443))
+
+When producing prometheus statistics the otel crate (0.19.0) now automatically appends "_total" which is unhelpful.
+
+This fix remove duplicated "_total_total" from our statistics.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3471
+
```

Comment:
Strange. I thought this was in 1.25.0. In fact it was. I just checked in the 1.25.0 changelog... Maybe the issue here is because it was cherry-picked into the 1.25.0 release? Anyway, this can be removed.

Reply:
oh i didnt know it was cherry picked. fixing, ty!

Reply:
fixed in e79130a5